src/clj/rems/application/model.clj

```diff
@@ -253,6 +253,7 @@
     {:permission :application.command/add-member}
     {:permission :application.command/assign-external-id}
     {:permission :application.command/change-resources}
+    {:permission :application.command/close}
```

Comment:
Hmm, does this actually give decider the permission to close applications, as well as the applicant? Is that what we want?

Reply:
It let's the "usual" behaviour of close through from the master workflow. So: applicant can close drafts, handler can close submitted & approved applications

Reply:
Ok, thanks!